resources/js/exts.js

```diff
@@ -0,0 +1,5 @@
+/* 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/resources/js/exts.js
+++ b/resources/js/exts.js
@@ -1,4 +1,4 @@
-/* 
+/*
 Extensions from coala-utils python library
 https://gitlab.com/coala/coala-utils/blob/master/coala_utils/Extensions.py 
 */
```